src/main/java/org/jvnet/hudson/test/SupportLogFormatter.java

```diff
@@ -1,210 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright (c) 2013, CloudBees, Inc.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-package org.jvnet.hudson.test;
-
-// COPIED from com.cloudbees.jenkins.support @ 2.42-SNAPSHOT
-
-// *********************************************
-// DO NOT INCLUDE ANY NON JDK CLASSES IN HERE.
-// IT CAN DEADLOCK REMOTING - SEE JENKINS-32622
-// *********************************************
-import java.io.PrintStream;
-import java.io.PrintWriter;
-import java.io.StringWriter;
-import java.util.HashSet;
-import java.util.Set;
-import java.util.logging.Formatter;
-import java.util.logging.LogRecord;
-// *********************************************
-// DO NOT INCLUDE ANY NON JDK CLASSES IN HERE.
-// IT CAN DEADLOCK REMOTING - SEE JENKINS-32622
-// *********************************************
-
-/**
- * Format log files in a nicer format that is easier to read and search.
- *
- * @author Stephen Connolly
- */
-class SupportLogFormatter extends Formatter {
-
-    private static long start = System.nanoTime();
-    static String elapsedTime() { // modified from original
```

Comment:
Ok, I think I get the DeltaSupportLogFormatter class, because this method got added here and hence is not in https://github.com/jenkinsci/lib-support-log-formatter/blob/master/src/main/java/io/jenkins/lib/support_log_formatter/SupportLogFormatter.java

Reply:
Correct. For the specialized use case of functional tests, I found it more helpful to show seconds elapsed since start of test, rather than absolute timestamps.